test/compare_seed_catalog.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import argparse
+import sys
+import os
+
+def run_brogue_tests(seed_catalog_file, extra_args, max_level):
+    
+    output_seed_catalog_file = f'output_seed_catalog_{max_level}.txt'
+    seed_catalog_cmd_args = f'--print-seed-catalog 1 100 {max_level}'
```

Comment:
idea: Given RNG cascades it may be enough to shorten this to maybe just 10 seeds for a much shorter test, if you feel it's valid?

Reply:
Hmm, my experience of trying to port bullet brogue to the new framework showed some subtleties in dungeon gen that only occurred on types of runics or similar. So I might go for 25 as a compromise.